AdyenActions/Components/SDK/TwintSDKActionComponent.swift

```diff
@@ -121,12 +120,21 @@ import Foundation
         }
 
         private func invokeTwint(app: TWAppConfiguration, action: TwintSDKAction) {
-
-            let error = twint.pay(
-                withCode: action.sdkData.token,
-                appConfiguration: app,
-                callback: configuration.callbackAppScheme
-            )
+            let error: Error? = {
```

Comment:
What do you think now?

Reply:
Now it's better.
I might throw another suggestion but it's up to you to decide if it's better or not.
Assuming both `twint.registeForUOF` and `twint.pay` have same signatures, this can be improved a bit:

```
let operation: (String, AppConfiguration, String) -> Error?
    
if action.sdkData.isStored {
    operation = twint.registerForUOF
} else {
    operation = twint.pay
}
    
if let error = operation(action.sdkData.token, app, configuration.callbackAppScheme) {
    handleShowError(error.localizedDescription)
    return false
}
```